habitat_baselines/utils/common.py

```diff
@@ -99,27 +102,40 @@ def __init__(
         self.action_activation = config.action_activation
         self.use_log_std = config.use_log_std
         self.use_softplus = config.use_softplus
+        self.use_std_param = config.use_std_param
         if config.use_log_std:
             self.min_std = config.min_log_std
             self.max_std = config.max_log_std
+            std_init = 0.0  # initialize std value so that exp(std) ~ 1
         else:
             self.min_std = config.min_std
             self.max_std = config.max_std
+            std_init = 1.0  # initialize std value so that std ~ 1
 
         self.mu = nn.Linear(num_inputs, num_outputs)
-        self.std = nn.Linear(num_inputs, num_outputs)
-
         nn.init.orthogonal_(self.mu.weight, gain=0.01)
         nn.init.constant_(self.mu.bias, 0)
-        nn.init.orthogonal_(self.std.weight, gain=0.01)
-        nn.init.constant_(self.std.bias, 0)
+
+        if self.use_std_param:
+            self.std = torch.nn.parameter.Parameter(
+                torch.randn(num_outputs) * 0.01 + std_init
+            )
+        else:
+            self.std = nn.Linear(num_inputs, num_outputs)
+            nn.init.orthogonal_(self.std.weight, gain=0.01)
+            nn.init.constant_(self.std.bias, std_init)
 
     def forward(self, x: Tensor) -> CustomNormal:
         mu = self.mu(x)
         if self.action_activation == "tanh":
             mu = torch.tanh(mu)
 
-        std = torch.clamp(self.std(x), min=self.min_std, max=self.max_std)
+        if self.use_std_param:
+            std = self.std
+        else:
+            std = self.std(x)
+
+        std = torch.clamp(std, min=self.min_std, max=self.max_std)
```

Comment:
Shouldn't we clamp after the exp since this would clamp the log std, not the std? I also think that clamping should be optional. It was not used in the Habitat 2.0 experiments.

Reply:
@ASzot  3 things : 
 - 1. We can either change the default for the std parameter or modify each config file. I think we should do it in subsequent change (probably does not need a PR)
 - 2. Do you agree with decreasing the entropy coefficient on all configs? I observed training collapses after some time if the entropy coefficient is too high. Do you agree with the change?
 - 3. The min_std and max_std in the exponential case correspond to `config.min_log_std` and `config.max_log_std` so the min and max are actually log_min and log_max so I think this is the correct behavior.

Reply:
i. Sounds good.
ii. Yes, I agree with decreasing the entropy coefficient.
iii. Ok I see.